nf_core/pipelines/lint/__init__.py

```diff
@@ -594,7 +593,7 @@ def run_linting(
     lint_obj._load_lint_config()
     lint_obj.load_pipeline_config()
 
-    if "nfcore_components" in lint_obj.lint_config and not lint_obj.lint_config["nfcore_components"]:
+    if lint_obj.lint_config and lint_obj.lint_config["nfcore_components"] is False:
```

Comment:
just a suggestion, I find it easier to read
```suggestion
    if lint_obj.lint_config and not lint_obj.lint_config["nfcore_components"]:
```